src/util/bitmap.cc

```diff
@@ -279,6 +279,27 @@ bool Bitmap::InterpolateBilinear(const double x, const double y,
   return false;
 }
 
+bool Bitmap::ExifCameraModel(std::string* camera_model) const {
+  // Read camera make and model
+  std::string make_str;
+  std::string model_str;
+  std::string focal_length;
+  *camera_model = "";
+  if (ReadExifTag(FIMD_EXIF_MAIN, "Make", &make_str)) {
```

Comment:
I think each if block here should have an else { return false } and in the negative case also set camera_model = ""